packages/sdk/src/middleware/middleware.ts

```diff
@@ -37,7 +38,16 @@ const actionToUri = (action: TSdkAction, projectKey?: string): string => {
   const service = requestBuilder[action.payload.service];
   if (action.payload.options) service.parse(action.payload.options);
 
-  return service.build();
+  return service.build({
+    // given `service=orderEdits` and given `applyOrderEditTo`, we build an apply endpoint
+    // given `service=orderEdits` and no `applyOrderEditTo`, we build an update endpoint
+    // https://docs.commercetools.com/api/projects/order-edits
+    applyOrderEdit:
+      action.payload.service === SUPPORTED_SERVICES.ORDER_EDITS &&
+      typeof action.payload.options?.applyOrderEditTo === 'string',
```

Comment:
```suggestion
    applyOrderEdit: getShouldApplyOrderEdit(action)
```

Somewhere above:

```js
function getShouldApplyOrderEdit(action) {
   return (
      action.payload.service === SUPPORTED_SERVICES.ORDER_EDITS &&
      typeof action.payload.options?.applyOrderEditTo === 'string'
   );
}
```

Generally I wonder what would happen if we demand to pass it in as an option on the payload? So `action.payload.options.applyOrderEditTo`. Would that work or is it against the API?

Reply:
no, we can not really demand it, unless we use unions.
But using unions wouldn't make that the much sense to implement since the consumer can't necessarily be TS

Reply:
Got yah. Thanks for clarifying. To combine with my point above. Code looks sharp, tests suberb. I'd subjectively not have the constants and slam the check into a small function.